src/Components/Components/src/IHandleAfterRender.cs

```diff
@@ -11,6 +13,7 @@ public interface IHandleAfterRender
         /// <summary>
         /// Notifies the component that it has been rendered.
         /// </summary>
-        void OnAfterRender();
+        /// <returns>A <see cref="Task"/> that represents the asynchronous event handling operation.</returns>
```

Comment:
This looks like a bad copy-paste

Reply:
I couldn't think of anything else to say here so went with the boiler plate. Suggestions?

Reply:
`asynchronous event handling operation` -> `asynchronous rendering operation`